5-Cookies/server/tests/utils/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { v4 as uuidv4 } from "uuid";
+import { Guid } from "../../../common/interfaces/Todo";
+
+export const createMockTodo = (text:string) => ({
+  text,
+  isFinished: false,
+  id: generateID(),
+  userID: generateID(),
```

Comment:
in my comment before about `chance.string` i meant there's also `chance.uuid`...